### PR TITLE
update editors

### PR DESCRIPTION
**Pull request by basti1302**
As discussed in the working group meeting yesterday, let's bring the editors up to date. The intention is to backport this to the level 2 branch.

***
<a href="https://pr-preview.s3.amazonaws.com/instana/trace-context/pull/534.html" title="Last updated on Jun 6, 2023, 7:13 PM UTC (179fe88)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/534/ddc2159...instana:179fe88.html" title="Last updated on Jun 6, 2023, 7:13 PM UTC (179fe88)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,8 +25,18 @@
       },
       {
         name: "Yuri Shkuro",
-        company: "Facebook",
+        company: "Meta",
```
Comment (basti1302) [inline index.html:28]: @yurishkuro FYI updating your employer's name if you are okay with that.

File: index.html [hunk 2]
```diff
@@ -25,8 +25,18 @@
...
         w3cid: "104074"
+      },
+      {
+        name: "J. Kalyana Sundaram",
+        company: "Microsoft",
+        w3cid: "??????"
```
Comment (basti1302) [inline index.html]: @kalyanaj The editor datasets all have the editor's w3c ID. I didn't find a way to get your ID from anywhere. Could you please provide it if you have a minute? Thanks in advance.

**Comment by basti1302** [comment]
AFAICT, everything else is up to date in terms of editors. Let me know if anything else needs to be changed.

File: index.html (hunk 1 above)
Comment (yurishkuro) [reply index.html:28]: +1

**Review by kalyanaj — APPROVED** [review]

**Comment by basti1302** [comment]
@SergeyKanzhelev FYI I have removed all companies/company URLs from former editors, as discussed.

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**